power_grid_model_c/power_grid_model/include/power_grid_model/calculation_parameters.hpp

```diff
@@ -222,6 +222,50 @@ struct ShortCircuitMathOutput {
     std::vector<ComplexValue<sym>> i_source;
 };
 
+template <typename T>
+concept symmetric_math_output_type = std::same_as<T, MathOutput<true>> || std::same_as<T, ShortCircuitMathOutput<true>>;
```

Comment:
i would like to replace the `<sym>` template with a tag relating to the calculation type but that requires an even larger overhaul (using `tag` structs instead of `bool sym`) and it's probably better to hold that off until after short circuit is done. this change at least scopes down the other overhaul